.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea/
 *.pyc
 *.iml
+logging/
+config_dev.yaml
+passfile.txt
```

Comment:
The configuration file I am using on my dev environment and the file holding my EGA credentials.  I put them in gitignore because I didn't want to upload them.